workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Android
+{
+    public static class AndroidDeviceInfo
+    {
+        public const string AndroidStudioEmulatorDefaultCallbackIp = "10.0.2.2";
+
+        private static bool isInitialised = false;
+        private static bool isEmulator = false;
```

Comment:
could be changed to an enum:
```
enum DeviceType = {
  UNKNOWN,
  PHONE,
  EMULATOR
}
```

that makes it easier to later add detection whether it's a phone or a tablet. Depending on that you often want to change how to display things.

Reply:
actually, I really want this to be changed to an enum :)